static/js/index.js

```diff
@@ -13,4 +13,11 @@ if (emblaRoot) {
     prevBtn?.addEventListener('click', carousel?.scrollPrev, false);
     nextBtn?.addEventListener('click', carousel?.scrollNext, false);
   });
-};
\ No newline at end of file
+};
+
+// Display cookie banner on user click.
+const cookieBttn = document.querySelector('.cookie-bttn');
+cookieBttn?.addEventListener('click', () => {
```

Comment:
The script was provided by HubSpot to display the cookie banner to allow users to adjust their preferences at any time.